src/freenet/node/Node.java

```diff
@@ -756,6 +746,8 @@ public void set(String val) throws InvalidConfigValueException {
 	
 	private boolean enableRoutedPing;
 
+	private boolean peersOffersDismissed;
```

Comment:
Is there a way to keep this functionality out of `Node`? It’s huge enough as it is…

Reply:
One option is to create a `configuration` package and create classes there that will contain groups of them. It makes no sense to move something separately, but if there is a lot ..